### ISSUE TITLE
Rise of Shadows?

### ISSUE THREAD

**Issue by TestamentCrux** [mtimkovich/hearthsounds#7]
Hi, I'm not sure if you're still working on this project, but I hope you are! I'd love to hear the new sounds from Rise of Shadows. I can't find any other online resource that reliably provides the card sounds. I just want you to know that you've done a great job, and people are still very interested in Hearthsounds! Thank you!

**Comment by mtimkovich** [comment]
Yeah, I'll be updating the sound database as soon as I can. It's so great to hear that you're enjoying HearthSounds!
